src/frontend/src/pages/myfeeds.js

```diff
@@ -15,94 +34,312 @@ const useStyles = makeStyles((theme) => ({
   },
 }));
 
+function HelpPopoverButton() {
+  const classes = useStyles();
+
+  return (
+    <PopupState variant="popover" popupId="help-popover">
+      {(popupState) => (
+        <div>
+          <IconButton
+            color="secondary"
+            classes={{ root: classes.button }}
+            {...bindTrigger(popupState)}
+          >
+            <HelpOutline />
+          </IconButton>
+          <Popover
+            {...bindPopover(popupState)}
+            anchorOrigin={{
+              vertical: 'bottom',
+              horizontal: 'center',
+            }}
+            transformOrigin={{
+              vertical: 'top',
+              horizontal: 'center',
+            }}
+          >
+            <Box p={2}>
+              <Typography align="center">
+                Please enter the web address of your blog&apos;s RSS or Atom feed.
+              </Typography>
+              <Typography align="center">
+                <em>
+                  (<a href="https://rss.com/blog/find-rss-feed/">Not sure how to find that?</a>)
+                </em>
+              </Typography>
+            </Box>
+          </Popover>
+        </div>
+      )}
+    </PopupState>
+  );
+}
+
+function DeleteDialogButton({ feed }) {
+  const classes = useStyles();
+  const [open, setOpen] = React.useState(false);
+
+  const handleClickOpen = () => {
+    setOpen(true);
+  };
+
+  const handleClose = () => {
+    setOpen(false);
+  };
+
+  const removeFeed = () => {
+    console.log(`Removing feed hosted at URL ${feed.url}`);
+    // TODO
+    window.location.reload(false);
+  };
+
+  return (
+    <div>
+      <IconButton classes={{ root: classes.button }} onClick={handleClickOpen}>
+        <Delete color="secondary" />
+      </IconButton>
+      <Dialog
+        open={open}
+        onClose={handleClose}
+        aria-labelledby="alert-dialog-title"
+        aria-describedby="alert-dialog-description"
+      >
+        <DialogTitle id="alert-dialog-title">{`Remove feed hosted at ${feed.url}?`}</DialogTitle>
```

Comment:
You might want to think about how you'll deal with really long URLs here.  I checked, and the longest one we have in the wiki is 87 characters (average is 51 characters).

Reply:
Posted an idea here: https://github.com/Seneca-CDOT/telescope/issues/945#issuecomment-609954691